index/scorch/scorch.go

```diff
@@ -295,12 +297,17 @@ func (s *Scorch) Batch(batch *index.Batch) (err error) {
 	// wait for analysis result
 	analysisResults := make([]*index.AnalysisResult, int(numUpdates))
 	var itemsDeQueued uint64
+	var totalAnalysisSize int
 	for itemsDeQueued < numUpdates {
 		result := <-resultChan
+		resultSize := result.Size()
+		atomic.AddUint64(&s.iStats.newSegBufBytesAdded, uint64(resultSize))
```

Comment:
Sorry for missing this before, but shouldn't this have been analysisBytesAdded, and below analysisBytesRemoved?